ci/build.bat

```diff
@@ -0,0 +1,30 @@
+@ECHO OFF
+setlocal enabledelayedexpansion
+
+IF NOT DEFINED B2_CI_VERSION (
+    echo You need to set B2_CI_VERSION in your CI script
+    exit /B 1
+)
+
+PATH=%ADDPATH%%PATH%
+
+SET B2_TOOLCXX=toolset=%B2_TOOLSET%
+
+IF DEFINED B2_CXXSTD (SET B2_CXXSTD=cxxstd=%B2_CXXSTD%)
+IF DEFINED B2_CXXFLAGS (SET B2_CXXFLAGS=cxxflags=%B2_CXXFLAGS%)
+IF DEFINED B2_DEFINES (SET B2_DEFINES=define=%B2_DEFINES%)
+IF DEFINED B2_ADDRESS_MODEL (SET B2_ADDRESS_MODEL=address-model=%B2_ADDRESS_MODEL%)
+IF DEFINED B2_LINK (SET B2_LINK=link=%B2_LINK%)
+IF DEFINED B2_VARIANT (SET B2_VARIANT=variant=%B2_VARIANT%)
+
+cd %BOOST_ROOT%
+
+IF DEFINED SCRIPT (
+    call libs\%SELF%\%SCRIPT%
```

Comment:
If `SELF` is guaranteed to be defined when `SCRIPT` is, then perhaps `IF NOT DEFINED SELF ( ...error...)` is redundant but could be documenting assert, or a comment.

Reply:
Which `IF NOT DEFINED SELF` do you mean?

Reply:
Sorry, I wrote bad English. I mean, adding `NOT DEFINED SELF` check is or may be redundant, but perhaps it would make it clear. I mean, `ci/build.bat` is used by AzP and AppVeyor, but only AppVeyor sets `SCRIPT`. I may be over-engineering here, please forget.